lib/scavenger_hunt/app/controllers/scavenger_hunt/players_controller.rb

```diff
@@ -14,7 +14,7 @@ def new
   end
 
   def show
-    redirect_to new_player_path unless current_player? && current_player.email.present?
+    redirect_to new_player_path unless current_player?
```

Comment:
I changed this because the player's email is not a required field. Therefore when a player signs up and doesn't put an email, they were never being directed to the "thank you" page with links to move further in the process (leaderboard, survey, etc.)